preupg/creator/ui_helper.py

```diff
@@ -195,6 +190,24 @@ def get_content_info(self):
             desc = get_user_input(settings.content_desc_text, any_input=True)
             self.content_dict['content_description'] = desc
 
+    @staticmethod
+    def _write_to_file_content(file_path, config):
```

Comment:
Ok, this doesn't save whatever data to a file, it receives a config object and saves it as a file. In this case, the code is fine.